docs/tuning-apps/fault-tolerance/supported.rst

```diff
@@ -0,0 +1,41 @@
+Supported fault tolerance techniques
+====================================
+
+Open MPI is a vehicle for research in fault tolerance and over the years
+provided support for a wide range of resilience techniques:
+
+* Current
+    * User Level Fault Mitigation techniques similar to
+      those implemented in FT-MPI.
+
+* Deprecated / no longer available
+    * Coordinated and uncoordinated process checkpoint and
+      restart. Similar to those implemented in LAM/MPI and MPICH-V,
+      respectively.
+    * |strikethru_start| Message logging techniques. Similar to those
```

Comment:
What's the purpose of the strikethrough, to indicate that these are deprecated? If that's the case, shouldn't line #12-14 also have a strikethru?

Reply:
> What's the purpose of the strikethrough, to indicate that these are deprecated? If that's the case, shouldn't line #12-14 also have a strikethru?

The strikethrough was in the original markdown, so I was carrying it through in my changes.  I believe that it was intended to indicate that those items had not just been deprecated, but had also been deleted.